src/app/dashboard/components/greeting/greeting.component.html

```diff
@@ -1,10 +1,13 @@
 <div class="greeting">
   <div class="content">
+
     <div class="header">
-      <span>{{currentTime()}}, <b>{{user.firstName}}</b>!</span>
+      <span>{{currentTime}}, <b>{{user.firstName}}</b>!</span>
     </div>
+
     <div class="sub-header">
       <span class="main-text">Take a look at your activities, standings, friends and more.</span>
     </div>
+
```

Comment:
3 and 11 lines - you can remove them :)
please double check all files and formatt html properly. thanks 🙃

Reply:
fixed